package.json

```diff
@@ -33,6 +34,7 @@
     "isstream": "^0.1.2",
     "mocha": "^7.0.0",
     "nyc": "^15.0.0",
-    "standard": "^14.3.1"
+    "standard": "^14.3.1",
+    "stream-array-es6": "^1.0.1"
```

Comment:
Have you thought about using [into-stream](https://www.npmjs.com/package/into-stream)? It's more popular and has more releases and more recent releases, so hopefully, it went through more quality checks.

Reply:
Probably it somehow showed up higher in my search results. I can switch